### ISSUE TITLE
shoudn't be able to change WP Report of Status "Publish" back to "Draft"?

### ISSUE THREAD

**Issue by mscholtz** [washingtontrails/vms#268]
VMS allows me to take a published WP report and change it back to draft.  Not sure this should be possible.  @WaBirder57 what say you?

Furthermore, if I had already filled in the crew message and that email had gone out, if I later change it back to published status, it will send the message again.  So even if we do allow setting it back to Draft, I think we need a mechanism to avoid repeating the email send.

**Comment by mscholtz** [comment]
also, very minor, but can we change the status to "Published" instead of "Publish"?  The latter belongs on a button, but not on a status value.

**Comment by WaBirder57** [comment]
I agree with both of these.  Once the report is Published it should not go back to a status of "Published".

**Comment by davisagli** [comment]
So once it's published should we not show the Edit button at all? Or can the leader still edit a published report?

**Comment by WaBirder57** [comment]
Right, don't show Edit since that is not an option we will support.

**Comment by davisagli** [comment]
Okay, the edit button no longer shows up for reports with a status of 'Publish'.

@mscholtz let me know when you want to change the value from Publish to Published so we can do it at the same time.

**Comment by mscholtz** [comment]
@davisagli done over here.  
FWIW, next time you don't have to wait for us.  It will allow you to save values that aren't in the picklist.

**Comment by davisagli** [comment]
@mscholtz this workflow rule needs to be updated too: https://cs9.salesforce.com/01QK00000001EjI ...There be other workflow rules or code that use "Publish" instead of "Published".

**Comment by davisagli** [comment]
All mentions of 'Publish' now changed to 'Published' in pyramid

**Comment by mscholtz** [comment]
fixed that WF, and don't see any other references to this value

**Comment by mscholtz** [comment]
waiting to test this since Completed WPs currently disappear from the VMS index, giving us a very brief window to view them after the report is published.  Alec is working on a change here.  @davisagli assigning to you temporarily, please send it back when completed WPs can be accessed

**Comment by davisagli** [comment]
Completed work parties should be accessible now
